sql/src/main/java/org/apache/druid/sql/calcite/planner/Calcites.java

```diff
@@ -100,6 +100,15 @@ public class Calcites
   public static final SqlReturnTypeInference
       ARG1_NULLABLE_ARRAY_RETURN_TYPE_INFERENCE = new Arg1NullableArrayTypeInference();
 
+  public static SqlReturnTypeInference complexReturnTypeWithNullability(ColumnType columnType, boolean nullable)
```

Comment:
Oh, I guess we could have defined this method on `OperatorBuilder`, similar to the others https://github.com/apache/druid/blob/master/sql/src/main/java/org/apache/druid/sql/calcite/expression/OperatorConversions.java#L366